experimental/federated_plugin/README.md

```diff
@@ -0,0 +1,20 @@
+# federated_plugin
+
+A Flutter plugin sample that shows how to implement federated plugin to fetch device location on different platforms.
+
+This sample is currently being built. Not all platforms and functionality are in place.
+
+## Goals for this sample
+
+* Shows how to develop a federated plugin which supports Android, iOS, Web & Desktop.
```

Comment:
nit: "web" and "desktop" don't need to be capitalized, since they're not proper nouns.